test/fixtures/url/default.txt

```diff
@@ -0,0 +1,93 @@
+doesn't break simple text
+.
+Hello World
+.
+<p>Hello World</p>
+.
+
+works with simple urls
+.
+http://flowdock.com
+.
+<p><a href="http://flowdock.com">http://flowdock.com</a></p>
+.
+
+works with text before the url
+.
+baz http://flowdock.com
+.
+<p>baz <a href="http://flowdock.com">http://flowdock.com</a></p>
+.
+
+works with text after the url
+.
+http://flowdock.com bar
+.
+<p><a href="http://flowdock.com">http://flowdock.com</a> bar</p>
+.
+
+works with text before and after the url
+.
+bar http://flowdock.com baz
+.
+<p>bar <a href="http://flowdock.com">http://flowdock.com</a> baz</p>
+.
+
+works with other markup before the url
+.
+* http://flowdock.com
+.
+<ul>
+<li><a href="http://flowdock.com">http://flowdock.com</a></li>
+</ul>
+.
+
+works with markup after the url
+.
+http://flowdock.com **baz**
+.
+<p><a href="http://flowdock.com">http://flowdock.com</a> <strong>baz</strong></p>
+.
+
+works with markup before the url
+.
+**baz** http://flowdock.com
+.
+<p><strong>baz</strong> <a href="http://flowdock.com">http://flowdock.com</a></p>
+.
+
+should not replace url in code block
+.
+`http://flowdock.com`
+.
+<p><code>http://flowdock.com</code></p>
+.
+
+should work in blockquotes
+.
+> http://flowdock.com
+.
+<blockquote>
+<p><a href="http://flowdock.com">http://flowdock.com</a></p>
+</blockquote>
+.
+
+should work with multiple urls in markdown
+.
+* http://flowdock.com
+* http://twitter.com
+.
+<ul>
+<li><a href="http://flowdock.com">http://flowdock.com</a></li>
+<li><a href="http://twitter.com">http://twitter.com</a></li>
+</ul>
+.
+
+should work with multiple urls in the same line
+.
+http://flowdock.com is cool http://twitter.com
+.
+<p><a href="http://flowdock.com">http://flowdock.com</a> is cool <a href="http://twitter.com">http://twitter.com</a></p>
+.
```

Comment:
I'd like to see a couple of samples for the broken cases with markdown rendering, like _emphasis_ and __double emphasis__ being converted inside links or broken nested parens in the link (and with surrounding parens).